tests/Doctrine/Tests/DBAL/Functional/Schema/SQLServerSchemaManagerTest.php

```diff
@@ -75,12 +73,9 @@ public function testDefaultConstraints()
         self::assertEquals('Doctrine rocks!!!', $columns['df_string_2']->getDefault());
         self::assertEquals('another default value', $columns['df_string_3']->getDefault());
         self::assertEquals(1, $columns['df_boolean']->getDefault());
-        self::assertSame($platform->getCurrentDateSQL(), $columns['df_current_date']->getDefault());
-        self::assertSame($platform->getCurrentTimeSQL(), $columns['df_current_time']->getDefault());
```

Comment:
During introspection, the string representation of the actual expression on the column doesn't necessarily match the expression in the schema definition. They only used to match because of the bug.

While the expression being deployed looks like `CONVERT(date, GETDATE())`, the introspected expression could be `CONVERT([date],getdate())` (type is quoted, no whitespace after the colon, lowerr-cased function name) or even [`CONVERT([date],getdate(),0)`](https://ci.appveyor.com/project/doctrine/dbal/build/job/75o41rjwc4b3u291#L274) depending on the platform version.

Reply:
This is already tested in `DefaultExpressionTest`.